docs/spire-quickstart.md

```diff
@@ -168,7 +168,7 @@ We want to replace the image with the Tornjak image: `tsidentity/tornjak-spire-s
       serviceAccountName: spire-server
       containers:
         - name: spire-server
-          image: tsidentity/tornjak-spire-server:1.0.0
+          image: ghcr.io/spiffe/tornjak-spire-server:1.0.1
```

Comment:
quick start uses 1.0.0 (https://github.com/spiffe/spire-tutorials/blob/master/k8s/quickstart/server-statefulset.yaml#L23) for now so let's stick to 1.0.0

Reply:
Fixed